### PR TITLE
Cart for football product

### PR DESCRIPTION
**Pull request by utkarshducoder**
please review my football cart page

live link: https://sportssite.netlify.app/html/card1.html

this page has following features:

- section: for hero image.
- table: for main documentation of the page.

f3
f4

### CODE REVIEW

File: football cart/card1.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          <!-- <button  style="border-radius: 20%; height: 30px; width: 35px;">-</button><br><br> -->
+        </div> <br> <button style="width: 80%; background-color: grey; color: white;height: 10%; font-size: 20px;">Remove From Cart</button><br><br>
+        <button style="width: 80%; height: 50%; background-color: grey; color: white;height: 10%; font-size: 20px;">Move To Wishlist</button>
+          </div>
+          
+        </div>
+      </div>
+    </td>
+ 
+
```
Comment (khushijohri2001) [inline football cart/card1.html]: correct the alignment and indentation

File: football cart/card1.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <table>
+
+      <td>
+        <div class="card-image">
+        
+          <div class="secondary-cart">
+            <h2 style="font-size: 32px;">Price Details</h2> <hr>
+            <h2 style="font-size: 30px;">Price(2 Details)Rs. 1,680</h2>
+            <h2 style="font-size: 30px;">Discount -Rs.1999</h2>
+            <h2 style="font-size: 30px;">Deleivery Charges -Rs.499</h2>
```
Comment (khushijohri2001) [inline football cart/card1.html]: Delivery spelling is wrong in delivery charges

**Review by khushijohri2001 — APPROVED** [review]

File: football cart/card1.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  }
+
+
+  .product-info{
+
+    display: flex;
+    flex-direction: column;
+ justify-content: center;
+ align-items: center;
```
Comment (i-am-a-codegirl) [inline football cart/card1.css]: do proper allignment

**Review by i-am-a-codegirl — APPROVED** [review]
Code looks good to merge just check my comments

**Pull request merged**